SimPEG/Examples/Downloads/SimPEGtemp/LdM_grav_obs.grv

```diff
@@ -0,0 +1,192 @@
+191				
```

Comment:
Nan good catch, I must have ran the example locally and forgot to delete the temp folder. Should we just had a line in his script to remove the downloads after run?